subiquity/models/keyboard.py

```diff
@@ -211,3 +211,14 @@ def set_keyboard(self, setting):
             else:
                 scale = os.environ.get('SUBIQUITY_REPLAY_TIMESCALE', "1")
                 run_command(['sleep', str(1/float(scale))])
+
+    def render(self):
```

Comment:
Seems like this part of refactor to support the reorg for bootloader  should have been a different PR.

Reply:
Hah yes, how did that get here!

Reply:
Oh wait, this is the change from #477 which is being reviewed separately I hope